app/post/controllers/post-detail-controller.js

```diff
@@ -274,4 +277,33 @@ function (
             Notify.showApiErrors(errorResponse);
         });
     };
+
+    $scope.addToSurvey = function () {
+       $location.path('/posts/create/' + post.form.id);
+    };
+
+    // Get duration since post was last updated
+    function getUpdateDuration() {
```

Comment:
Moment can do this for you: moment().fromNow(); .. it also does the locatization for you more sanely.
post card directive wraps that with a switcher to change to long date format for things that happened a while ago
```
function formatDates() {
                var created = moment($scope.post.created),
                    now = moment();

                if (now.isSame(created, 'day')) {
                    $scope.displayTime = created.fromNow();
                } else {
                    $scope.displayTime = created.format('LL');
                }
                $scope.displayTimeFull = created.format('LLL');
            }
```
Something similar to that would be better than hacking our own together.